roles/ovirt-host-deploy-ovirt-metrics/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+galaxy_info:
```

Comment:
We decided to store those roles internally and don't publish them on galaxy. So please remove this file.

Reply:
Ah, so I see you want them in galaxy based on:

 https://bugzilla.redhat.com/show_bug.cgi?id=1494947

Hmm, so we need to properly document it, first.

Reply:
@rgolangh I don't think we need it in Galaxy. Roy can you please update here if we do and if it is not necessary, please update bugzilla. Thanks.

Reply:
I talked to Roy and Galaxy is not required